exo-add/features/exoservice-es6-mongodb.feature

```diff
@@ -54,7 +54,7 @@ Feature: scaffolding an ExoService written in ES6, backed by MongoDB
           - user.updated
 
       docker:
-        link:
+        dependencies:
```

Comment:
My thought was that seeing as these are Docker images that the service requires to have containers running for (could be specifying a certain Docker image version), it would make sense to keep these dependencies listed in the docker section, along with anything else Docker related such as ports to be published, etc. 

What do you think?